web/views/manga/views.py

```diff
@@ -7,36 +7,39 @@
 from django.views.generic import ListView
 from django.views.generic.detail import DetailView
 
-from api.models import Manga, Volume
+from api.models import Manga, Volume, Edition
 from web.forms import MangaForm
 
 
 class MangaDetailView(DetailView):
-    template_name = "web/detail.html"
+    template_name = "web/manga_volumes.html"
     context_object_name = "manga"
     model = Manga
 
     def get_context_data(self, **kwargs):
         context = super(MangaDetailView, self).get_context_data(**kwargs)
         manga = context["manga"]
-        open_vol_request = self.request.GET.get("volume")
-        open_vol = -1
-        try:
-            open_vol = int(open_vol_request)
-        except BaseException:
-            # If the volume the user provided is not a number, just default to the non tankobon chapters
-            open_vol = -1
-
-        volumes = Volume.objects.filter(manga=manga, absolute_number__gte=0).order_by(
+        edition = self.request.GET.get("edition")
+
+        editions = list(Edition.objects.filter(manga=manga).values_list("name", flat=True))
```

Comment:
Exclude the current edition from this. This is so that the drop down does not include the current edition that is being viewed. Just nicer ig.